### DISCUSSION TITLE
Remove all options from virtual select

### DISCUSSION THREAD

**Discussion by it-groemp** [sa-si-dev/virtual-select#340] [Q&A]
Hi everyone,
I have to populate one dropdown based on another dropdown. I am calling an ajax function to get the values. My issue is I am not able to remove the options which are added previously. Can someone help me with the same.

Thank you in advance

**Comment by lucy-at-mc** [comment]
I'm implementing exactly the same functionality. Here's how to remove everything from the selector:

`document.querySelector('#sample-select').setOptions([]);`

I love this library, but the lack of clarity about how to basic things like these are pretty annoying.
